content/messaging/agent-chat-api/v3.2/rtm-reference/index.mdx

```diff
@@ -3693,7 +3693,10 @@ It returns the initial state of the current Agent.
 | `application`                       | No       | `object`  |                                                                                                                                  |
 | `application.name`                  | No       | `string`  | Application name                                                                                                                 |
 | `application.version`               | No       | `string`  | Application version                                                                                                              |
-| `away`                              | No       | `bool`    | Controls whether the connection should be considered to be in the away state by the auto-away feature. Defaults to false.        |
+| `away` __*__                        | No       | `bool`    | When `true`, the connection is set to the `away` state. Defaults to `false`.  |
+
+__*__ You can use the `away` param to **prevent assigning chats** to Agents with `not_accepting_chats` **after random reconnections**. When an Agent logs in with `away: true`, the connection is immediately recognized
```

Comment:
>**prevent assigning chats** to Agents with `not_accepting_chats` **after random reconnections**

this might suggest that the chat assignment can happen even when the agent set `not_accepting_chats` with `set-routing-status` which is not the case

how about this?
__*__ You can use the `away` param to  **prevent assigning chats** to Agents **after random reconnections** when their status was set to `not_accepting_chats` by the auto-away feature.